_episodes/06-exporting-data.md

```diff
@@ -1,68 +1,70 @@
 ---
 title: "Exporting data"
 teaching: 10
-exercises: 10
+exercises: 5
 questions:
 - "How can we export data from spreadsheets in a way that is useful for downstream applications?"
 objectives:
 - "Store spreadsheet data in universal file formats."
 - "Export data from a spreadsheet to a CSV file."
-- "Add a column to a dataframe to indicate the filename."
 keypoints:
 - "Data stored in common spreadsheet formats will often not be read correctly into data anlysis software, introducing errors into your data."
 - "Exporting data from spreadsheets to formats like CSV or TSV puts it in a format that can be used consistently by most programs."
+- "TSV format is more appropriate than CSV for data containing commas, like free-response survey data."
 ---
 
-## Spreadsheet storage formats
+Storing the data you're going to work with for your analyses in Excel
+default file format (`*.xls` or `*.xlsx` - depending on the Excel
+version) isn't a good idea. Why?
 
-Excel and other spreadsheet applications store the data in their own proprietary formats. 
-This is fine if you only intend to use the data in the same application that it was created in, but it
-can make it difficult to open the spreadsheet files in other applications that you may want to use to 
-process the data.
+- Because it is a proprietary format, and it is possible that in
+the future, technology won’t exist (or will become sufficiently
+rare) to make it inconvenient, if not impossible, to open the file.
 
-In future versions of Excel Microsoft may make changes to the format, so that existing files are not even readable by the latest versions of Excel.
+- Other spreadsheet software may not be able to open files
+saved in a proprietary Excel format.
 
+- Different versions of Excel may handle data
+differently, leading to inconsistencies.
 
-The .xlsx format used by versions of Excel since 2007 is in fact a folder of compressed XML files. 
-You can rename a .xlsx file as a .zip file and extract the individual files. 
-Amongst other things there will be an XML file for each of the tabs in your spreadsheet.
+- Finally, more journals and grant agencies are requiring you
+to deposit your data in a data repository, and most of them don't
+accept Excel format. It needs to be in one of the formats
+discussed below.
 
-You could load this XML file into a text editor and read the data in it. But XML is a very verbose data format, designed originally for 
-transferring data between one computer application and another. You may be able to read it but it would not be in the nice tabular format
-that you see when it is loaded into Excel.
+- The above points also apply to other formats such as open data formats used by LibreOffice / Open Office. These formats are not static and do not get parsed the same way by different software packages.
```

Comment:
remove mention of Open Office. There are several issues with it, and we shouldn't recommend its use at this time.